src/base/static/components/form-fields/types/datetime-field.js

```diff
@@ -22,19 +30,46 @@ class DatetimeField extends Component {
         timeFormat={this.props.timeFormat}
         renderInput={(datetimeProps, openCalendar) => {
           return (
-            <input
-              type="text"
-              ref={input => {
-                this.inputRef = input;
-              }}
-              className="datetime-field__input"
-              onFocus={openCalendar}
-              placeholder={this.props.placeholder || "Select a date"}
-              value={
-                this.props.value &&
-                moment(this.props.value).format(this.props.displayFormat)
-              }
-            />
+            <div className="datetime-field__input-wrapper">
+              {this.props.ongoingLabel && (
+                <label
```

Comment:
Can we abstract this checkbox into its own CheckboxSelection atom? Perhaps we can create a new file under `components/atoms/selections.js`.

Reply:
I agree, but do you think it would be better to just migrate all field types to atoms in one PR? I think I'd rather not have some field types as atoms and others still housed in the `form-field/types` directory.

I can open a second PR for this.

Reply:
As we discussed earlier, I think it is easier to create a Checkbox or CheckboxSelection atom now, and start using that. Whenever we are creating selections, `components/atoms/selections.js` should be the first place to look.

I'm wary about adding yet another checkbox implementation to our codebase, but if you think it's better to do this later, that's fine too.